content/kubermatic/v2.17/tutorials_howtos/upgrading/upgrade_from_2.16_to_2.17/chart_migration/_index.en.md

```diff
@@ -0,0 +1,325 @@
++++
+title = "Migrating to the Operator"
+date = 2021-04-21T11:09:15+02:00
+weight = 30
+
++++
+
+This chapter outlines a migration plan for the Kubermatic Operator, starting from having KKP installed
+using the old `kubermatic` Helm chart using Helm 2.x. or 3.x.
+
+{{% notice warning %}}
+Before performing the migration, make sure to back up all Kubermatic resources!
+{{% /notice %}}
+
+Make sure that you also have recent etcd snapshots for all user clusters.
+
+#### Overview
```

Comment:
I would put a warning mentioning that chart installations are not supported any longer at KKP 2.17 and that clusters have to be imperatively migrated. Wdyt?

Reply:
Sure I can put it.

Reply:
done